sci-geosciences/grass/grass-7.8.3.ebuild

```diff
@@ -0,0 +1,273 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite"  # bug 572440
+WANT_AUTOCONF="2.1"
+WX_GTK_VER=3.0
+
+inherit autotools desktop eapi7-ver python-single-r1 toolchain-funcs wxwidgets xdg flag-o-matic
+
+MY_PM=${PN}$(ver_cut 1-2 ${PV})
+MY_PM=${MY_PM/.}
+MY_P=${P/_rc/RC}
+
+DESCRIPTION="A free GIS with raster and vector functionality, as well as 3D vizualization"
+HOMEPAGE="https://grass.osgeo.org/"
+SRC_URI="https://grass.osgeo.org/${MY_PM}/source/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/7.8.3"
+KEYWORDS="amd64 ~ppc ~x86"
+IUSE="blas cxx fftw geos lapack liblas mysql netcdf nls odbc opencl opengl openmp png postgres readline sqlite threads tiff truetype X"
+
+RDEPEND="${PYTHON_DEPS}
+	>=app-admin/eselect-1.2
+	$(python_gen_cond_dep '
+		|| (
+			dev-python/numpy[${PYTHON_MULTI_USEDEP}]
+		)
+	')
+	media-libs/libprojectm
```

Comment:
Please check all dependencies listed here which we link against, for whether they have a subslot defined. If they do, subscribe to it so we get rebuilt if their ABI breaks with ':='.